cloudapi/config.go

```diff
@@ -33,128 +33,28 @@ import (
-	AggregationWaitPeriod types.NullDuration `json:"aggregationWaitPeriod" envconfig:"K6_CLOUD_AGGREGATION_WAIT_PERIOD"`
-
-	// If aggregation is enabled, but the collected samples for a certain AggregationPeriod after AggregationPushDelay has passed are less than this number, they won't be aggregated.
-	AggregationMinSamples null.Int `json:"aggregationMinSamples" envconfig:"K6_CLOUD_AGGREGATION_MIN_SAMPLES"`
-
-	// If this is enabled and a sub-bucket has more than AggregationMinSamples HTTP trails in it, they would all be
-	// aggregated without attempting to find and separate any outlier metrics first.
-	// IMPORTANT: This is intended for testing purposes only or, in extreme cases, when the result precision
-	// isn't very important and the improved aggregation percentage would be worth the potentially huge loss
-	// of metric granularity and possible masking of any outlier samples.
-	AggregationSkipOutlierDetection null.Bool `json:"aggregationSkipOutlierDetection" envconfig:"K6_CLOUD_AGGREGATION_SKIP_OUTLIER_DETECTION"`
-
-	// If aggregation and outlier detection are enabled, this option specifies the
-	// number of HTTP trails in a sub-bucket that determine which quartile-calculating
-	// algorithm would be used:
-	// - for fewer samples (between MinSamples and OutlierAlgoThreshold), a more precise
-	//   (i.e. supporting interpolation), but also more computationally-heavy sorting
-	//   algorithm will be used to find the quartiles.
-	// - if there are more samples than OutlierAlgoThreshold in the sub-bucket, a
-	//   QuickSelect-based (https://en.wikipedia.org/wiki/Quickselect) algorithm will
-	//   be used. It doesn't support interpolation, so there's a small loss of precision
-	//   in the outlier detection, but it's not as resource-heavy as the sorting algorithm.
-	AggregationOutlierAlgoThreshold null.Int `json:"aggregationOutlierAlgoThreshold" envconfig:"K6_CLOUD_AGGREGATION_OUTLIER_ALGO_THRESHOLD"`
-
-	// The radius (as a fraction) from the median at which to sample Q1 and Q3.
-	// By default it's one quarter (0.25) and if set to something different, the Q in IQR
-	// won't make much sense... But this would allow us to select tighter sample groups for
-	// aggregation if we want.
-	AggregationOutlierIqrRadius null.Float `json:"aggregationOutlierIqrRadius" envconfig:"K6_CLOUD_AGGREGATION_OUTLIER_IQR_RADIUS"`
-
-	// Connection or request times with how many IQRs below Q1 to consier as non-aggregatable outliers.
-	AggregationOutlierIqrCoefLower null.Float `json:"aggregationOutlierIqrCoefLower" envconfig:"K6_CLOUD_AGGREGATION_OUTLIER_IQR_COEF_LOWER"`
-
-	// Connection or request times with how many IQRs above Q3 to consier as non-aggregatable outliers.
-	AggregationOutlierIqrCoefUpper null.Float `json:"aggregationOutlierIqrCoefUpper" envconfig:"K6_CLOUD_AGGREGATION_OUTLIER_IQR_COEF_UPPER"`
+	PushRefID                       null.String        `json:"pushRefID" envconfig:"K6_CLOUD_PUSH_REF_ID"`
```

Comment:
Dropping the comments here is a big no ;)

Reply:
I see the same problem in several changes, I need to restore them. Hopefully, they aren't a problem for the real size.

Reply:
Also, in general, I am not sure we should optimize these config fields for size, that probably won't help a lot with the actual k6 performance... Rather, we should optimize them for human readability, grouping related fields together...

Reply:
This is why I would get a benchmark before do any progress here. Because, if we don't get any improvement then we can just close this PR without any addition.